TransferMethod/Sources/ListTransferMethodController.swift

```diff
@@ -95,6 +95,13 @@ final class ListTransferMethodController: UITableViewController {
         return cell
     }
 
+    private func showEdit(_ indexPath: IndexPath) {
+        let coordinator = HyperwalletUI.shared.updateTransferMethodCoordinator(presenter.sectionData[indexPath.row],
+                                                                               true,
+                                                                               parentController: self)
+        coordinator.navigate()
+    }
+
```

Comment:
No need to add this logic here, Nandana will work on this ticket. For your testing you can directly update ViewController and add another row in there to Update Transfer Method

Reply:
Sure Jaspreet